roles/gateway_export_database/README.md

```diff
@@ -14,9 +14,9 @@ Requirements
     4) Click [Delete Events] when prompted to confirm
   
      ```  
-* ssh connect to remote source gateway as user ssgconfig using encrypted password. 
-    Specify source gateway's hostname/ip in the hosts file [gateway_mysql_source] section.
-    ```
+* Must be able to ssh from the Ansible controller to the source gateway as user ssgconfig. 
+* Specify source gateway's hostname/ip in the hosts.yml file [gateway_mysql] section.
+    ``` 
     [gateway_mysql_source]
```

Comment:
should also be rename to [gateway_mysql]?

Reply:
This section will need to be updated when the host file improvement PR has been merged.